frame/support/src/inherent.rs

```diff
@@ -260,7 +264,7 @@ mod tests {
 	fn check_inherents_works() {
 		let block = Block::new(
 			Header::new_from_number(1),
-			vec![Extrinsic { function: Call::Test(CallTest::Something) }],
+			vec![Extrinsic { function: Call::Test2(CallTest2::Something) }],
```

Comment:
```suggestion
			vec![
				Extrinsic { function: Call::Test2(CallTest2::Something) },
				Extrinsic { function: Call::Test(CallTest::Something) },
			],
```

Please do not change the semantics of a test.

Reply:
My bad, I had changed it to the smallest block for which the inherent check worked, but I recognize now that wasn't what the test did before.